components/server/src/workspace/workspace-starter.ts

```diff
@@ -1563,6 +1565,11 @@ export class WorkspaceStarter {
             sysEnvvars.push(ev);
         }
 
+        const orgIdEnv = new EnvironmentVariable();
+        orgIdEnv.setName("GITPOD_WORKSPACE_DEFAULT_IMAGE");
+        orgIdEnv.setValue(await this.configProvider.getDefaultImage(workspace.organizationId));
+        sysEnvvars.push(orgIdEnv);
+
         const spec = new StartWorkspaceSpec();
```

Comment:
**suggestion**: Definitely not the best line to comment about this, but since setting a non-reachable or non-functional workspace image could break UX for many org users, we could at least inform users why their workspace is broken. 

From [relevant discussion](https://gitpod.slack.com/archives/C05H5UQBW6Q/p1695028797614049?thread_ts=1695026341.413039&cid=C05H5UQBW6Q) (internal):

> Otherwise, users would be left in blank if they don't have organization settings access. I don't think users can guess there's something wrong with the custom image, right?

What do you think of adding an error alert in the workspace start, when a custom default image, and ideally we could check if it's accessible or not? If we can't check if the image is reachable, we could tone down the error as we do with the last editor warning and warn users that there's a custom set image that could be responsible for this workspace error.

| Error alert | Warning alert |
|-|-|
| <img width="448" alt="Frame 1885" src="https://github.com/gitpod-io/gitpod/assets/120486/50059b69-134a-4861-8488-b50077ff1d34"> | <img width="448" alt="Frame 1886" src="https://github.com/gitpod-io/gitpod/assets/120486/e5e5fbc7-a92a-40d0-a25d-c506c2c1d4c5"> |

☝️ Copy for the warning could be improved.
☝️ We use that area for the latest editor warning, which could be nice if we replaced. In the future, we could hide the latest editor warning all together or tone down the message.

Reply:
@mustard-mh can we do something here?